packages/cyberstorm/src/components/DropDown.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/DropDown.module.css";
+
+import * as RadixDropDown from "@radix-ui/react-dropdown-menu";
+import { SelectItemProps } from "./SelectItem";
+
+type DropDownProps = {
+  colorScheme?: "default" | "defaultDark" | "primary";
+  defaultOpen?: boolean;
+  content?: ReactNode[];
+  trigger: ReactNode;
+};
+
+export const DropDown: React.FC<DropDownProps> = React.forwardRef(
+  (props, ref) => {
+    const { colorScheme, defaultOpen, content, trigger } = props;
+
+    return (
+      <div className={styles.root} {...props} ref={ref}>
+        <RadixDropDown.Root defaultOpen={defaultOpen}>
+          <RadixDropDown.Trigger asChild>
+            {React.isValidElement(trigger)
+              ? React.cloneElement(trigger, props)
+              : trigger}
+          </RadixDropDown.Trigger>
+
+          <RadixDropDown.Portal>
+            <RadixDropDown.Content
+              align="start"
+              sideOffset={8}
+              className={`${styles.content} ${getContentStyle(colorScheme)}`}
+            >
+              {parseContent(content, colorScheme)}
+            </RadixDropDown.Content>
+          </RadixDropDown.Portal>
+        </RadixDropDown.Root>
+      </div>
+    );
+  }
+);
+
+DropDown.displayName = "DropDown";
+DropDown.defaultProps = {
+  defaultOpen: false,
+  colorScheme: "default",
+};
+
+const parseContent = (
+  content: ReactNode[] | undefined,
+  colorScheme: "default" | "defaultDark" | "primary" | undefined
+) => {
+  const props = { colorScheme } as SelectItemProps;
```

Comment:
Since content is now just `ReactNode`s, this row seems pointless. It could be inlined on L55.